pkgs/misc/drivers/hplip/3.16.11.nix

```diff
@@ -142,8 +142,6 @@ pythonPackages.buildPythonApplication {
 
     mkdir -p $out/etc/sane.d/dll.d
     mv $out/etc/sane.d/dll.conf $out/etc/sane.d/dll.d/hpaio.conf
-
-    rm $out/etc/udev/rules.d/56-hpmud.rules
```

Comment:
How is the udev rule installed on NixOS?

Reply:
Currently I'm simply adding `pkgs.hplipWithPlugins` to `config.services.udev.packages`.